languages/vinyl.pot

```diff
@@ -47,67 +47,201 @@ msgid "Vinyl Audio requires PHP 7.4 or higher."
+#: src\utils\time.js:41
+msgid "minute"
+msgstr ""
+
+#: src\utils\time.js:42
+msgid "minutes"
+msgstr ""
+
+#: src\utils\time.js:45
+msgid "second"
+msgstr ""
+
+#: src\utils\time.js:46
+msgid "seconds"
+msgstr ""
+
+#: src\labels\create-labels.js:5
+msgid "audio player"
+msgstr ""
+
+#: src\labels\create-labels.js:6
+msgid "video player"
+msgstr ""
+
+#: src\labels\create-labels.js:7
+msgid "volume"
+msgstr ""
+
+#: src\labels\create-labels.js:8
+msgid "seek"
+msgstr ""
+
+#: src\labels\create-labels.js:9
+msgid "closed captions"
+msgstr ""
+
+#: src\labels\create-labels.js:13
+msgid "playback time"
+msgstr ""
+
+#: src\labels\create-labels.js:14
+msgid "media loading"
+msgstr ""
+
+#: src\labels\create-labels.js:15
+msgid "settings"
+msgstr ""
+
+#: src\labels\create-labels.js:16
+msgid "audio tracks"
+msgstr ""
+
+#: src\labels\create-labels.js:17
+msgid "quality"
+msgstr ""
+
+#: src\labels\create-labels.js:21
+msgid "play"
+msgstr ""
+
+#: src\labels\create-labels.js:22
+msgid "pause"
+msgstr ""
+
+#: src\labels\create-labels.js:23
+msgid "mute"
 msgstr ""
 
-#: block.json
-#. title
-msgid "title"
+#: src\labels\create-labels.js:24
+msgid "unmute"
 msgstr ""
 
-#: block.json
-#. description
-msgid "description"
+#: src\labels\create-labels.js:25
+msgid "start airplay"
 msgstr ""
 
-#: block.json
-#. keywords
-msgid "keywords"
+#: src\labels\create-labels.js:26
+msgid "stop airplay"
 msgstr ""
 
-#: src/audio/edit/caption.tsx:28
-#: src/audio/edit/caption.tsx:80
+#: src\labels\create-labels.js:27
+msgid "start casting"
+msgstr ""
+
+#: src\labels\create-labels.js:28
+msgid "stop casting"
+msgstr ""
+
+#: src\labels\create-labels.js:29
+msgid "enter fullscreen mode"
+msgstr ""
+
+#: src\labels\create-labels.js:30
+msgid "exit fullscreen mode"
+msgstr ""
+
+#: src\labels\create-labels.js:31
+msgid "enter picture in picture mode"
+msgstr ""
+
+#: src\labels\create-labels.js:32
+msgid "exit picture in picture mode"
+msgstr ""
+
+# number of seconds to seek forward.
+#: src\labels\create-labels.js:36
+msgid "seek forward %s second"
+msgid_plural "seek forward %s seconds"
+msgstr[0] ""
+msgstr[1] ""
+
+# number of seconds to seek backward.
+#: src\labels\create-labels.js:47
+msgid "seek back %s second"
+msgid_plural "seek back %s seconds"
+msgstr[0] ""
+msgstr[1] ""
+
+#: src\labels\create-labels.js:55
+msgid "seek to live"
+msgstr ""
+
+#: src\labels\create-labels.js:56
+msgid "playing live"
+msgstr ""
+
+#: src\audio\edit.tsx:152
+msgid "Settings"
+msgstr ""
+
+#: src\audio\edit.tsx:155
+msgid "Loop"
+msgstr ""
+
+#: src\audio\edit.tsx:175
+msgid "Browser default"
+msgstr ""
+
+#: src\audio\edit.tsx:177
+msgid "Auto"
+msgstr ""
+
+#: src\audio\edit.tsx:180
+msgid "Metadata"
+msgstr ""
+
+#: src\audio\edit.tsx:209
+msgid "Audio caption text"
+msgstr ""
+
+#: src\audio\edit\caption.tsx:28
+#: src\audio\edit\caption.tsx:80
 msgid "Add caption"
 msgstr ""
 
-#: src/audio/edit/caption.tsx:29
+#: src\audio\edit\caption.tsx:29
```

Comment:
@erikyo is this an intentional change to how `@wp-blocks/make-pot` writes the paths?

Reply:
He confirmed, this is intentional.

Reply:
yes, but even if it was intentional, I decided to reverse the slashes again because looking at the gettext doc the 'old' way seems to be the correct one (or more correct since they both work)

i think the problem I was reported with make-json was related to the fact that i was not normalising the paths. now it should all be 'fixed', please update to 1.3.2 and run again the script. And thank you again for the report!!!

Reply:
Upgraded and fixed.